# differential drive steering commands. 

std_msgs/Header header

#accel command
float32 accel 
float32 vel   

# set curvature OR angular_velocity
float32 curvature
#float32 angular_velocity

float32 velL
float32 velR

float32 pwmL
float32 pwmR

